.github/workflows/client.yml

```diff
@@ -12,22 +12,50 @@ on:
       - "solidity/dashboard/**"
   pull_request:
   workflow_dispatch:
+    inputs:
+      environment:
+        description: 'Environment for workflow execution'
+        required: false
+        default: 'dev'
+      upstream_builds:
+        description: 'Upstream builds'
+        required: false
+      ref:
```

Comment:
According to RFC-18 this property should be named `upstream-ref`.

Reply:
Right, this will be delivered in following PRs:
https://github.com/keep-network/ci#3
https://github.com/keep-network/run-workflow/pull/2
https://github.com/keep-network/notify-workflow-completed#1
https://github.com/keep-network/keep-core/pull/2466
https://github.com/keep-network/keep-ecdsa/pull/786
https://github.com/keep-network/tbtc/pull/793
https://github.com/keep-network/tbtc-dapp/pull/395
https://github.com/keep-network/tbtc.js/pull/123